Packs/Core/Playbooks/playbook-Netcat_Makes_or_Get_Connections_README.md

```diff
@@ -0,0 +1,58 @@
+This playbook is designed to handle the following alerts:
+
+- Netcat makes or gets connections
+
+The playbook executes the following stages:
+
+Analysis:
+
+- Investigate the IP and Domain  reputation
+- Search previous similar alerts
+
+Remediation:
+
+- Handles malicious alerts by terminating the causality process.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* CoreIOCs
+* CortexCoreIR
+* CortexCoreXQLQueryEngine
+
+### Scripts
+
+* SearchIncidentsV2
```

Comment:
The command name is not "SearchAlertsV2"?

Reply:
it's the same command but the validation requires this one